examples/docker-compose.yml

```diff
@@ -5,6 +5,8 @@ services:
     init: true
     network_mode: host
```

Comment:
network_mode: host is still required?

Reply:
It's not required but it's easier if you're going to use the stone for connecting an external session. So I kept these examples with host networking enabled and those with the gem disabled (because here you cannot use the same netldi for 2 different containers).

Reply:
Which is a must during development, right. Otherwise connections from Windows VAST + gembuilder or Jadeite would be refused, only those from the gem container would be allowed. For deployment we must not forget to disable it.